src/internal/operators/switchMap.ts

```diff
@@ -114,7 +114,7 @@ export function switchMap<T, R, O extends ObservableInput<any>>(
               // it ends up being smaller than using the map operator.
               (innerValue) => subscriber.next(resultSelector ? resultSelector(value, innerValue, outerIndex, innerIndex++) : innerValue),
               () => {
-                // The inner has completed. Null out the inner subcriber to
+                // The inner has completed. Null out the inner subscribe to
```

Comment:
Sigh.
```suggestion
                // The inner has completed. Null out the inner subscriber to
```

Reply:
Since there's `innerSubscriber` variable that gets set to `null` after this comment, `subscriber` is the right one.